apps/not-arrived/translations/src/en/fields.json

```diff
@@ -47,6 +47,10 @@
       }
     }
   },
+  "case-id-number": {
+    "label": "Case ID Number",
+    "hint": "(Enter the case ID found on your visa decision letter.)"
```

Comment:
Should it be capital for case?

Reply:
They use the same wording else where in the service and its a lower case:

"case-id": {
      "label": "Case ID number",
      "hint": "(Enter the case ID found on your visa decision letter.)"
},